x-pack/plugins/observability_solution/entity_manager/server/routes/enablement/disable.ts

```diff
@@ -13,6 +13,39 @@ import { uninstallBuiltInEntityDefinitions } from '../../lib/entities/uninstall_
 import { canDisableEntityDiscovery } from '../../lib/auth/privileges';
 import { EntityDiscoveryApiKeyType } from '../../saved_objects';
 
+/**
+ * @openapi
+ * /internal/entities/managed/enablement:
+ *   delete:
+ *     description: Disable managed (built-in) entity discovery. This stops and deletes the transforms, ingest pipelines, definitions saved objects, and index templates for this entity definition, as well as the stored API key for entity discovery management.
+ *     tags:
+ *       - management
+ *     parameters:
+ *       - in: query
+ *         name: deleteData
+ *         description: If true, delete all entity data in the managed indices
+ *         required: false
+ *         schema:
+ *           type: boolean
+ *           default: false
+ *     responses:
+ *       200:
+ *         description: OK - validate disable result in response body
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 success:
+ *                  type: boolean
+ *                  example: false
+ *                 reason: 
+ *                  type: string
+ *                  example: user_not_authorized
+ *                 message:
+ *                  type: string
+ *                  example: Current Kibana user does not have the required permissions to disable entity discovery
```

Comment:
not authorized [now returns 403](https://github.com/elastic/kibana/pull/189423)

Reply:
nice - i must have been on a slightly old branch - thanks.